content/en/actions/private_actions/run_script.md

```diff
@@ -86,14 +86,19 @@ In your workflow or app, configure the action to use the `runPredefinedScript` w
 
 {{< img src="service_management/par-script-variables.png" alt="The two levels of variables inside the runner." style="width:80%;" >}}
 
+## Large image
+
+If you want to use tools like [Python](https://www.python.org/), SSH, [AWS CLI](https://aws.amazon.com/cli/), [Terraform](https://developer.hashicorp.com/terraform/cli/commands) or the [gcloud CLI](https://docs.cloud.google.com/sdk/docs/install) you can use the `gcr.io/datadoghq/private-action-runner:v{{< private-action-runner-version "private-action-runner" >}}-large` image instead of the default image.
```

Comment:
```suggestion
If you want to use tools like [Python](https://www.python.org/), SSH, [AWS CLI](https://aws.amazon.com/cli/), [Terraform](https://developer.hashicorp.com/terraform/cli/commands), or the [gcloud CLI](https://docs.cloud.google.com/sdk/docs/install) you can use the `gcr.io/datadoghq/private-action-runner:v{{< private-action-runner-version "private-action-runner" >}}-large` image instead of the default image.
```
Please reformat the inline links into numbered links, like the other ones in the document. The translation process requires numbered links.